misc/dbt-materialize/dbt/include/materialize/macros/relations/columns_spec_ddl.sql

```diff
@@ -19,7 +19,7 @@
 {% macro materialize__get_table_columns_and_constraints() -%}
   {{ exceptions.warn(
         """
-        Materialize does not support constraints.
+        Materialize only supports non-null assertions constraints.
```

Comment:
Grammar here is wonky.

Reply:
Can we make this macro smarter and only warn if you try to use a constraint whose type is not `not_null`?